transformer/kubernetes/argocdtransformer.go

```diff
@@ -152,8 +153,26 @@ func (t *ArgoCD) setupEnhancedIR(oldir irtypes.IR, projectName string) irtypes.E
 		return common.MakeStringDNSSubdomainNameCompliant(fmt.Sprintf("%s-%s", projectName, baseName))
 	}
 	appName := p(baseAppName)
-	ir.ArgoCDResources = irtypes.ArgoCDResources{
-		Applications: []irtypes.Application{{Name: appName}},
+	destNamespace := qaengine.FetchStringAnswer(
+		common.ConfigTransformersKubernetesArgocdNamespaceKey,
+		"Enter the namespace where argocd pipeline to be deployed",
+		[]string{"If this is not relevant to you then give an empty string to use the default value for it."},
+		"",
+		nil,
+	)
+	if destNamespace == "" {
+		ir.ArgoCDResources = irtypes.ArgoCDResources{
+			Applications: []irtypes.Application{{
+				Name: appName,
+			}},
+		}
+	} else {
+		ir.ArgoCDResources = irtypes.ArgoCDResources{
+			Applications: []irtypes.Application{{
+				Name:          appName,
+				DestNamespace: destNamespace,
+			}},
+		}
```

Comment:
we don't need an if/else since in Golang the default for a variable/struct field `DestNamespace` of type `string` is the empty string  `""`

Reply:
@HarikrishnanBalagopal done 👍🏽